addons/sale/migrations/13.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Payam Yasaie <https://www.tashilgostar.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_field_renames = [
+    ('product.attribute', 'product_attribute',
+     'type', 'display_type'),
+    ('product.attribute.custom.value', 'product_attribute_custom_value',
+     'attribute_value_id', 'custom_product_template_attribute_value_id'),
```

Comment:
wrong, `attribute_value_id` and `custom_product_template_attribute_value_id` have different models

instead, save value doing column rename and in post migration do:
```python
openupgrade.logged_query(
        env.cr, """
        UPDATE product_attribute_custom_value pacv
        SET custom_product_template_attribute_value_id = ptav.id
        FROM product_attribute_value pav
        JOIN product_template_attribute_value ptav
            ON ptav.product_attribute_value_id = pav.id
        WHERE pacv.{} = pav.id
            AND pacv.custom_product_template_attribute_value_id IS NULL
        """.format(openupgrade.get_legacy_name('attribute_value_id'))
    )
```